tests/playbooks/tests_states.yml

```diff
@@ -47,3 +47,32 @@
             state: down
     - include_tasks: tasks/assert_device_absent.yml
     - include_tasks: tasks/assert_profile_absent.yml
+
+    # test case (downing a non existent profile)
+    # I can set down and remove a non existent profile
+    # This will be rescued until issue 188 is fixed
+    - name: Verify that the play fails with the expected error message
+      block:
+        - name: Set down non existent profile
+          include_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: non_existent
+                type: ethernet
```

Comment:
Does this work without the type?

Reply:
Yes.
Really sorry I forgot to remove this , I was using this to test rescue block in the beginning and I didn't change it when I changed the rest . I'll fix this right away .